blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -1,13 +1,53 @@
 import React from "react";
 import PropTypes from "@arc-fusion/prop-types";
-import { useContent } from "fusion:content";
-import { useFusionContext } from "fusion:context";
+import getTranslatedPhrases from "fusion:intl";
+import getProperties from "fusion:properties";
+import { useContent, useEditableContent } from "fusion:content";
+import { useComponentContext, useFusionContext } from "fusion:context";
 
-import { LazyLoad, isServerSide, videoPlayerCustomFields } from "@wpmedia/engine-theme-sdk";
-import { imageRatioCustomField } from "@wpmedia/resizer-image-block";
-import { LargePromoPresentation } from "@wpmedia/shared-styles";
+import { LazyLoad, localizeDateTime } from "@wpmedia/engine-theme-sdk";
+import {
+	Conditional,
+	Grid,
+	HeadingSection,
+	Icon,
+	Image,
+	Link,
+	MediaItem,
+	Stack,
+	formatURL,
+	getImageFromANS,
+	isServerSide,
+	Overline,
+	Heading,
+	Paragraph,
+	Date as DateDisplay,
+	formatAuthors,
+	Attribution,
+	Separator,
+	Video,
+} from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-large-promo";
 
 const LargePromoItem = ({ customFields, arcSite }) => {
+	const {
+		imageOverrideURL,
+		playVideoInPlace,
+		showByline,
+		showDate,
+		showDescription,
+		showHeadline,
+		showImage,
+		showOverline,
+		showVideoLabel,
+		showImageOrVideoLabel,
+		imageOrVideoLabelText,
+		aspectRatio,
+		// shrinkToFit,
```

Comment:
was the shrinkToFit added? It's currently commented out?

Reply:
shrinkToFit has not been used even in V1.

Reply:
`shrinkToFit` used to change the style of the `VideoWrap` component to reduce it's width to maintain the viewport height for some reason.  Simply setting the max-height would already confine the video (they don't stretch like images...) and we show the background in the Video Frame.  It's kind of a deprecated value, IMO.  It's always "true" and it's hidden from the user.

Reply:
if it's truly not used, the custom field can also be removed. @nschubach has more experience investigating this than me so I'd defer to him on it. Currently, shrinkToFit is in the custom fields though https://github.com/WPMedia/arc-themes-blocks/pull/1411/files#diff-f42cfdfa85e045ac26e5a58cc1b4bf5fe6a0ddeade356dbe710eb98526f1df38R348

Reply:
yeah, the field can be removed